### PR TITLE
fixing url-names at ModelControllerBuilder

### PR DESCRIPTION
**Pull request by jdiego**
The url_name attribute on _register_action_endpoint, except the create endpoint, points to primary key attribute name ( self._model_pk_name, instance to the model name. Another change: use lower case to url_name.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -5,9 +5,9 @@ build-backend = "flit_core.buildapi"
 [tool.flit.metadata]
 module = "ninja_extra"
 dist-name = "django-ninja-extra"
-author = "Ezeudoh Tochukwu"
-author-email = "tochukwu.ezeudoh@gmail.com"
-home-page = "https://github.com/eadwinCode/django-ninja-extra"
+author = "Diego Saraiva"
```
Comment (eadwinCode) [inline pyproject.toml:8]: @jdiego Do you want to be the author of the project? 😅

**Review by eadwinCode — COMMENTED** [review]
Thanks for fixing these

File: pyproject.toml (hunk 1 above)
Comment (jdiego) [reply pyproject.toml:8]: Excuse me... I think I made a mess... I'm really sorry... I edited the wrong pyproject.yml... sorry about that, really sorry... I aml fixing it now

**Comment by jdiego** [comment]
@eadwinCode I fix the pyproject. Again, Sorry. Now I will correct on fork ....

File: pyproject.toml (hunk 1 above)
Comment (eadwinCode) [reply pyproject.toml]: That's fine. No hard feelings

**Pull request merged**